torch/csrc/jit/script/compiler.h

```diff
@@ -233,6 +233,15 @@ TORCH_API Value* emitBuiltinCall(
   // otherwise it will return nullptr if the builtin is not found.
   bool required);
 
+struct TypeInfo {
```

Comment:
I don't think we want to make this change. See the conversation in this PR that initially made a similar change, and then was reverted. 

https://github.com/pytorch/pytorch/pull/13142